ref-tests/Cargo.toml

```diff
@@ -6,7 +6,7 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-candid = "0.7"
+candid = "0.7.1"
```

Comment:
and here

Reply:
Since the rest here seem to specify minor version too, I think it's ok

Reply:
Same as above.